client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningCheck.tsx

```diff
@@ -174,42 +173,31 @@ const ContactQuestioningCheck: React.FC<Props> = (props: Props): JSX.Element =>
-                        <Grid item xs={5}>
+                        <FieldName xs={5} fieldName={OCCUPATION_LABEL}/>
+                        <Grid item xs={7}>
                         <Controller
                             control={control}
                             name={`form[${index}.${InteractedContactFields.OCCUPATION}]`}
                             defaultValue={interactedContact.occupation}
                             render={(props) => {
-                                return(
-                                    <FormControl>
-                                        <RadioGroup
-                                            {...props}
-                                            aria-label={OCCUPATION_LABEL}
-                                            >
-                                            {
-                                                occupations.map((occupation) => {
-                                                    return (
-                                                        <FormControlLabel
-                                                            value={occupation}
-                                                            key={occupation}
-                                                            control={
-                                                                <Radio
-                                                                    disabled={isFieldDisabled}
-                                                                    color='primary'
-                                                                    onChange={(event) => {
-                                                                        props.onChange(event.target.value);
-                                                                    }}
-                                                                />
-                                                            }
-                                                            label={occupation}
-                                                        />
-                                                    )
-                                                })
-                                            }
-                                        </RadioGroup>
-                                    </FormControl>
-                                )
+                                return (<FormControl variant='outlined' className={classes.occupationSelect}>
+                                    <Select
+                                        {...props}
+                                        placeholder={OCCUPATION_LABEL}
+                                        onChange={(event) => {
+                                            props.onChange(event.target.value)
+                                        }}>
+                                        {
+                                            occupations.map((occupation) => (
```

Comment:
Is occupation guaranteed to be empty? What happenes if the get /occupation API call fails?

Reply:
basically, in the radio group they also didn't check the occupations, so I kept it the same.

Reply:
🤷

Reply:
fixed anyway